js/render-random-offers.js

```diff
@@ -0,0 +1,52 @@
+import {getRandomOffers} from './data.js';
+
+const offerCardTemplate = document.querySelector('#card')
+  .content
+  .querySelector('.popup');
```

Comment:
Давай уберем переносы строки единообразия ради
```suggestion
const offerCardTemplate = document.querySelector('#card').content.querySelector('.popup');
```